inlineplz/interfaces/__init__.py

```diff
@@ -5,6 +5,16 @@
 
 from inlineplz.interfaces.github import GitHubInterface
 
+STASH_SUPPORTED = False
+try:
+    from inlineplz.interfaces.stash import StashInterface
+    STASH_SUPPORTED = True
+except ImportError:
+    pass
+
```

Comment:
`prospector: pylint: Import "from inlineplz.interfaces.stash import StashInterface" should be placed at the top of the module (wrong-import-position)`